foodfuncs.py

```diff
@@ -1,403 +1,305 @@
-        print("Failed to connect and insert data", error)
-
+        execute_insert(instructions_query, instructions_data)
+        execute_insert(ingredients_query, ingredient_list)
+        execute_insert(recipes_query, recipes_data)
+        execute_insert(rec_ing_quan_query, rec_ing_quan_list)
+        print("the inserts were successful")
+    else:
+        print(f"This recipe already exists at id {recipe_id}.")
 
-def is_recipe_duplicate(name):
-     # WORKS
-    """input: name of recipe that user is trying to insert
-        output: true if duplicate, false if not
+def pickrecipe():
     """
-    # reads reclist, loops through entries and separates out recipe names to compare to name
-    file = open("reclist.txt")
-    recipes = file.read().split("\n")
-    file.close()
-    for item in recipes:
-        n = item.split(", ")
-        if n[1] == name:
-            return True
-    return False
-    # try:
-    #     conn = create_connection("recipes.db")
-    #     cur = conn.cursor()
-    #     dup = cur.execute("SELECT id FROM recipes WHERE recipe_name = ?", (name,),).fetchall()
-    #     if dup:
-    #         return True
-    #     else: 
-    #         return False
-    # except sqlite3.Error as error:
-    #     print("There was an error: ", error)
-    # finally:
-    #     if conn:
-    #         conn.close()
-    #         print("the connection is closed.")
-
-def is_ingredient_duplicate(name):
-    # WORKS
-    """input: name of ingredient that user is trying to insert
-        output: true if duplicate, false if not
+    Input: none
+    Finds the highest recipe id number, and randint picks within the range 1 - max 
+    Output: A dictionary as follows: 
+    recipe: "", ingredients: {x: amnt,}, instructions: "", difficulty: ""
+    OR False if no recipes to choose from
     """
-    # reads inglist, loops through entries and separates out ingredient names to compare to name
-    file = open("inglist.txt")
-    ingredients = file.read().split("\n")
-    file.close()
-    for item in ingredients: 
-        n = item.split(", ")
-        if n[1] == name:
-            return True
-    return False
-    # try:
-    #     conn = create_connection("recipes.db")
-    #     cur = conn.cursor()
-    #     dup = cur.execute("SELECT id FROM ingredients WHERE ingredient_name = ?", (name,),).fetchall()
-    #     if dup:
-    #         return True
-    #     else:
-    #         return False
-    # except sqlite3.Error as error:
-    #     print("There was an error: ", error)
-    # finally:
-    #     if conn:
-    #         conn.close()
-    #         print("the connection is closed.")
+    ids = rec_ing_instruc_new_ids()
+    greatest_recipe = ids[0]
+    if greatest_recipe > 0:
+        greatest_recipe = greatest_recipe -1
+    else:
+        # no recipes to choose from
+        return False
+    try:
+        full_recipe = {}
+        pick = (random.randint(0, greatest_recipe),)
+
+        recipe_query = """SELECT 
```

Comment:
I think this would look cleaner if the queries were grouped together at the top of the function, and then you call on all of them